.github/workflows/coverage-check.yml

```diff
@@ -22,13 +22,41 @@ jobs:
         uses: onbjerg/foundry-toolchain@v1
         with:
           version: nightly
+      
+      - name: Install dependencies
+        run: sudo apt-get install lcov
 
       - name: Get development branch coverage
         id: coverage-development
         run: |
           cd packages/contracts
+          
+          # generates lcov.info
+          forge coverage --report lcov
+
+          # Foundry uses relative paths but Hardhat uses absolute paths.
+          # Convert absolute paths to relative paths for consistency.
+          sed -i -e 's/\/.*solidity.//g' lcov.info
+
+          # Merge lcov files
+          lcov \
+              --rc lcov_branch_coverage=1 \
+              --add-tracefile lcov.info \
+              --output-file merged-lcov.info
+
+          # Filter out node_modules, test, and mock files
+          lcov \
+              --rc lcov_branch_coverage=1 \
+              --remove merged-lcov.info \
+              --output-file filtered-lcov.info \
+              "*node_modules*" "*test*" "*mock*" "*scriptscripting*"
```

Comment:
What is `scriptscripting` ?

Reply:
good catch! its a type it should be `*script*` and its meant to exclude the foundry deploy scripts

Reply:
fix in here;
https://github.com/ubiquity/ubiquity-dollar/pull/637/commits/13f762c75a5b60714d1270cbeaf88ba6d6da5253